.github/workflows/release-build-zip-file.yml

```diff
@@ -2,14 +2,20 @@ name: 'Release: Build ZIP file'
 on:
   workflow_dispatch:
     inputs:
-      ref:
-        description: 'By default the zip file is generated from the branch the workflow runs from, but you can specify an explicit reference to use instead here (e.g. refs/tags/tag_name or refs/heads/release/x.x). The resulting file will be available as an artifact on the workflow run.'
+      branch:
+        description: 'Source branch to use. Defaults to the branch the workflow is run from.'
         required: false
         default: ''
       skip_verify:
-        description: 'Skip the PR verification step (default: false) pass true to skip'
-        required: false
-        default: 'false'
+        description: 'Skip the PR verification step.'
```

Comment:
The description is outdated. How about something like `Skip verification steps.`?

Reply:
Addressed in a0c7d97964b6b02f960dc46b09e53fbd9eb62628.